src/lib/emailHelper.js

```diff
@@ -0,0 +1,6 @@
+// Return true if given email is a valid email, false either.
+module.exports.validate = function (email) {
```

Comment:
Maybe it's a personal opinion, but I feel that `isValid` is a better name for this function, as the return value is a boolean. `validate` implies that we are doing more things than juste testing if the value is correct.